### PR TITLE
Clean various parts of Sirius Components

### PR DESCRIPTION
**Pull request by sbegaudeau**

### CODE REVIEW

File: backend/sirius-web-view/.classpath [hunk 1]
```diff
@@ -22,6 +21,29 @@
...
+	</classpathentry>
+	<classpathentry kind="src" output="target/test-classes" path="target/generated-test-sources/test-annotations">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="ignore_optional_problems" value="true"/>
+			<attribute name="m2e-apt" value="true"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
```
Comment (gcoutable) [inline backend/sirius-web-view/.classpath:47]: Is this needed?

File: backend/sirius-web-view-edit/.classpath [hunk 2]
```diff
@@ -22,6 +21,29 @@
...
+	</classpathentry>
+	<classpathentry kind="src" output="target/test-classes" path="target/generated-test-sources/test-annotations">
+		<attributes>
+			<attribute name="optional" value="true"/>
+			<attribute name="maven.pomderived" value="true"/>
+			<attribute name="ignore_optional_problems" value="true"/>
+			<attribute name="m2e-apt" value="true"/>
+			<attribute name="test" value="true"/>
+		</attributes>
+	</classpathentry>
```
Comment (gcoutable) [inline backend/sirius-web-view-edit/.classpath:47]: Is this needed?

File: backend/sirius-web-view/.classpath (hunk 1 above)
Comment (sbegaudeau) [reply backend/sirius-web-view/.classpath:47]: That's the configuration of all our projects, I suspect that it has been overwritten by the EMF plugin generator.

File: backend/sirius-web-view-edit/.classpath (hunk 2 above)
Comment (sbegaudeau) [reply backend/sirius-web-view-edit/.classpath:47]: That's the configuration of all our projects, I suspect that it has been overwritten by the EMF plugin generator.

**Review by gcoutable — APPROVED** [review]

**Pull request merged**